app/styles/identity.css

```diff
@@ -39,11 +39,21 @@
     padding: 4px 15px;
 }
 
+.btn_div{
```

Comment:
NIT: is it necessary to tie a tag name in the class name? `_div`
May be, .btn-container ?

It is better to use kebab-case when come to css class names.

Reply:
thank you for your suggestion's,
I changed the class name to .btn-container and used kebab-case.